### ISSUE TITLE
Cannot connect to google cloud, receive ssh Error

### ISSUE THREAD

**Issue by reducedMatrice** [GoogleCloudPlatform/cloud-code-intellij#3146]
(Please ensure you are running the latest version of Cloud Code for IntelliJ with _Help > Check for Updates_)
- Cloud Code for IntelliJ version: 23.6.1-api-version-222
- Google Cloud SDK version: 436.0.0
- OS: Windows 10 10.0

**What did you do?**
I wanted to deploy my project to app engine in the google cloud, I tried 2 ways to do so. One was to use CLI run 'gcloud init', after I login with google account in the web page, going back to CLI 
 I received an error "ERROR: gcloud crashed (SSLError): HTTPSConnectionPool(host='oauth2.googleapis.com', port=443): Max retries exceeded with url: /token (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1129)')))"
The other way was to use the extension on Webstorm, and when tried to login, the web pages showed login successfully but I still fail to login. 
**What did you expect to see?**
I think it should be a login successfully result when I want to deploy my project, but it said 'no active account', so I was not log in.
**What did you see instead?**
I think the ssh error could probably be caused by my VPN, but I cannot access google account or services without VPN in my country, so I have to keep the VPN. Also I tried many other IPs but all failed. 
(screenshots are helpful)
_Feel free to deviate from this template as needed, especially if you are submitting a feature request._
![image](https://github.com/GoogleCloudPlatform/cloud-code-intellij/assets/111624273/a130591b-3652-402f-b8e1-be480256b83d)

**Comment by etanshaul** [comment]
hi @reducedMatrice . 

> The other way was to use the extension on Webstorm, and when tried to login, the web pages showed login successfully but I still fail to login.

Could you provide a few more details around this. What did you see happen after logging in from the plugin?  You said that it still failed to login, so I'm curious what you saw happen.

**Comment by reducedMatrice** [comment]
Hello Etan,

Thank you for responding. Here are the steps I took.
1.Go to the navigator bar, Tools - Google Cloud Code - Upload to Cloud
Source Repositories
[image: image.png]
2. After I selected my Google account, I agreed on Google Cloud  SKD to
access my account.
[image: image.png]

3. This page shows, telling me already authenticated. Meanwhile, nothing
happened in the webstorm and when I repeat step 1, it still needs me to
select Google Account and login.
[image: image.png]

Since what I received from the Google web page is exactly the same as I
tried authentication via CLI, I think Webstorm also failed to log in to
Google Account due to the same reason.

Etan Shaul ***@***.***> 于2023年6月27日周二 02:33写道：

**Comment by XiaoWen1227** [comment]
hey @reducedmatrice,
 did you solve this issue? i gotta same here.

**Comment by lautaropacellaDR** [comment]
Having the same issues when trying to connect a production cloud run to another cloud run. Anyone has solved this?

**Comment by adityaa30** [comment]
I had similar issue - https://unix.stackexchange.com/questions/577839/ssh-connection-freezes-on-4g-hotspot-but-not-on-any-wifi helped me. 

TLDR;
```bash
echo "ServerAliveInterval 115" >> ~/.ssh/config
```
